MAUI/ListView/swiping.md

```diff
@@ -0,0 +1,628 @@
+---
+layout: post
+title: Swiping in .NET MAUI ListView control | Syncfusion
+description: Learn here all about Swiping support in Syncfusion .NET MAUI ListView (SfListView) control, its elements and more.
+platform: MAUI
+control: SfListView
+documentation: ug
+---
+
+# Swiping in .NET MAUI ListView (SfListView)
+
+## Overview
+
+The `SfListView` allows swiping items to do custom actions such as deleting the data, adding the data, editing the data, etc. To enable swiping, set the `SfListView.AllowSwiping` property to `true`. Swipe views are displayed when swiping from left to right or right to left (for `Horizontal` orientation, top to bottom or bottom to top) on the item. 
+
+It provides customizable swipe templates for swiping on left and right sides. You can restrict the layout of swipe view up to a certain position when swiping the item by setting the `SfListView.SwipeThreshold` property. You can set size of the swipe views by setting the `SfListView.SwipeOffset` property.
```

Comment:
It provides customizable swipe templates for swiping on the left and right sides. You can restrict the layout of the swipe view up to a certain position when swiping the item by setting the `SfListView.SwipeThreshold` property. You can set the size of the swipe views by setting the `SfListView.SwipeOffset` property.

Reply:
resolved